scripts/package.json

```diff
@@ -11,6 +11,6 @@
   "dependencies": {
     "path": "^0.12.7",
     "solc": "^0.4.24",
-    "web3": "^1.0.0-beta.34"
+    "web3": "^1.0.0-beta2"
```

Comment:
I think it is not right to revert to web3 `1.0.0-beta2`. Maybe try to update to `1.0.0-beta.36`?

Reply:
Oh, sorry, yes. Will change to latest

Reply:
@varasev done